data-sources/sourcecred/output/sourcecred-101.csv

```diff
@@ -1,24 +1,24 @@
 week_end,sourcecred_score,discord_score,discourse_score,github_score
-2022-05-29,0.2561973274979917,0.2561973274979917,0,0
-2022-06-05,0.02705804176657996,0.02705804176657996,0,0
-2022-06-12,0.01438271916111869,0.01438271916111869,0,0
-2022-06-19,0.18455591053408704,0.11676897649905345,0.06778693403503361,0
-2022-06-26,0.6945104722039811,0.02349180440565425,0.6710186677983269,0
-2022-07-03,6.7605371584102265,0.11814879738668434,6.642388361023542,0
-2022-07-10,0.6853148097228,0.013473218006127592,0.6718415917166725,0
-2022-07-17,0.09259937596784236,0.016583225908784388,0.07601615005905797,0
-2022-07-24,0.24067716725208596,0.1523588277127924,0.08831833953929355,0
-2022-07-31,0.015391391191652877,0.015391391191652877,0,0
+2022-05-29,0,0,0,0
```

Comment:
@johnmark13 Looks like we have a scenario where the upper-case ETH address was added in SourceCred.  So we will probably go through the identities already registered in SourceCred, and make them all lower-case?

Added an issue for it here:  https://github.com/nation3/nationcred-instance/issues/130

Reply:
Or, modify the equality check to ignore case? We should be able to just parse the ledger and update - but that would not prevent uppercase being added in the future?

Reply:
@johnmark13 We should probably merge https://github.com/nation3/nationcred-instance/pull/131 and then re-run the file generation in this branch?